lib/commands/abstract-command.js

```diff
@@ -168,6 +178,31 @@ class AbstractCommand {
         }
     }
 
+    _remindsIfNewVersion(doDebug, callback) {
+        httpClient.request({
+            url: `${CONSTANTS.NPM_REGISTRY_URL_BASE}/${CONSTANTS.APPLICATION_NAME}/latest`,
+            method: CONSTANTS.HTTP_REQUEST.VERB.GET
+        }, 'GET_NPM_REGISTRY', doDebug, (err, response) => {
+            if (err) {
+                Messenger.getInstance().error(`Failed to get the latest version for ${CONSTANTS.APPLICATION_NAME} from NPM registry.\n${err}\n`);
+            } else {
+                const latestVersion = JSON.parse(response.body).version;
+                if (packageJson.version !== latestVersion) {
+                    const semVers = packageJson.version.split('.').map(x => parseInt(x, 10));
```

Comment:
semver module has some of the functionality build in. you can use major and minor function instead of parsing youself.
for example:

semver.major('2.1.1') => 2
semver.minor('2.1.1') => 1